packages/edit-site/src/components/sidebar/color-panel.js

```diff
@@ -3,63 +3,54 @@
  */
 import { __experimentalPanelColorGradientSettings as PanelColorGradientSettings } from '@wordpress/block-editor';
 import { __ } from '@wordpress/i18n';
-
-/**
- * Internal dependencies
- */
-import {
-	BACKGROUND_COLOR,
-	GRADIENT_COLOR,
-	LINK_COLOR,
-	TEXT_COLOR,
-} from '../editor/utils';
+import { STYLE_PROPERTY, LINK_COLOR } from '@wordpress/blocks';
 
 export default ( {
 	context: { supports, name },
 	getProperty,
 	setProperty,
 } ) => {
 	if (
-		! supports.includes( TEXT_COLOR ) &&
-		! supports.includes( BACKGROUND_COLOR ) &&
-		! supports.includes( GRADIENT_COLOR ) &&
+		! supports.includes( 'color' ) &&
```

Comment:
These keys are the ones that were updated in the server (`gutenberg_experimental_global_styles_get_support_keys` changes). This way we use the common convention for the styles properties everywhere.